components/collapsible-panel/collapsible-panel.js

```diff
@@ -343,10 +346,16 @@ class CollapsiblePanel extends SkeletonMixin(FocusMixin(RtlMixin(LitElement))) {
 		if (this._intersectionObserver) this._intersectionObserver.disconnect();
 	}
 
+	firstUpdated(changedProperties) {
+		super.firstUpdated(changedProperties);
+		this._button = this.shadowRoot.querySelector('.d2l-collapsible-panel-opener');
```

Comment:
Do we need to hold references to these elements? If `render` ever returns a result such that Lit renders a new button or content element, these references will no longer be valid. Also I might have missed it, but I don't see `this._button` referenced anywhere.

Reply:
That's a good point and I tend to agree. I think the Lit way to do this safely would be using the [ref directive](https://lit.dev/docs/templates/directives/#ref). But IMO the slight performance gain from holding these references doesn't seem worth it.